### PR TITLE
User Feed Queries for Logged in Users - Pagination!

### PR DESCRIPTION
**Pull request by wack-overflow**
This PR implements the calls made within the app to populate a user's threads and replies.

The big thing here is that it supports **PAGINATION** using instagram's `max_id` query param (renamed to `cursor`)

Please rip this PR apart if there is a better way you'd prefer for this type of implementation! ❤️

### CODE REVIEW

**Review by junhoyeo — APPROVED** [review]
Thank you so much @wack-overflow 
\+ If you tell me ur Threads username will be happy to shoutout 😉

File: threads-api/src/threads-api.ts [hunk 1]
```diff
@@ -351,7 +355,12 @@ export class ThreadsAPI {
   _getAppHeaders = () => ({
     'User-Agent': `Barcelona ${LATEST_ANDROID_APP_VERSION} Android`,
     'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8',
-    ...(this.token ? { Authorization: `Bearer IGT:2:${this.token}` } : undefined),
+    ...(this.token && { Authorization: `Bearer IGT:2:${this.token}` }),
+  });
+
+  _getInstaHeaders = () => ({
```
Comment (junhoyeo) [inline threads-api/src/threads-api.ts:361]: 👍👍👍

File: threads-api/src/threads-api.ts [hunk 2]
```diff
@@ -133,6 +133,10 @@ interface UserIDQuerier<T extends any> {
   (username: string, userID: string, options?: AxiosRequestConfig): Promise<T>;
 }
 
+interface PaginationUserIDQuerier<T extends any> {
```
Comment (junhoyeo) [inline threads-api/src/threads-api.ts:136]: Awesome Type Wrapper 👍

**Comment by wack-overflow** [comment]
> If you tell me ur Threads username will be happy to shoutout 😉

@junhoyeo hey sounds good!  It's here: https://www.threads.net/@wack.overflow

Cheers 🍻

**Comment by wack-overflow** [comment]
Although @junhoyeo looks like I just got the ban hammer LOL

I think paginating is not making them very happy over at threads HQ

**Comment by junhoyeo** [comment]
Fixed some small things in #153

**Comment by junhoyeo** [comment]
> Although @junhoyeo looks like I just got the ban hammer LOL

Omg I'm so sorry to hear that 😭  Hope you get your account recovered soon...

**Pull request merged**